db/db_test_util.cc

```diff
@@ -671,6 +671,21 @@ std::string DBTestBase::Get(int cf, const std::string& k,
   return result;
 }
 
+std::string DBTestBase::Get(const std::string& k, PinnableSlice* v) {
+  ReadOptions options;
+  options.verify_checksums = true;
+  std::string result;
+  Status s = dbfull()->Get(options, dbfull()->DefaultColumnFamily(), k, v);
+  if (s.IsNotFound()) {
+    result = "NOT_FOUND";
+  } else if (!s.ok()) {
+    result = s.ToString();
```

Comment:
Do you think we can have them not mixed? The convention in the code base is for the return value to be Status.